arangod/V8Server/V8DealerFeature.cpp

```diff
@@ -702,7 +705,19 @@ void V8DealerFeature::copyInstallationFiles() {
 
     size_t copied = 0;
 
-    auto filter = [&eslintPath, &binDirectory,
+    auto doFilterPath = [](std::string_view normalizedPath,
+                           std::string_view filterPath) -> bool {
+      if ((normalizedPath.size() >= filterPath.size() &&
+           normalizedPath.compare(normalizedPath.size() - filterPath.size(),
+                                  filterPath.size(), filterPath) == 0)) {
+        // filter it out!
+        return true;
+      }
+      return false;
+    };
+
+    auto filter = [&eslintPath, &uiNodeModulesPath, &doFilterPath,
```

Comment:
As far as I can see, the `eslintPath` should not be necessary any more.
At least the directory `js/node/node_modules/eslint` does not exist anymore in devel.

Reply:
replaced old outdated path with other one now: 04c150b33cd17d9e5d47cadae791bf4745c928df